src/test/groovy/com/github/vkorobkov/jfixtures/JFixturesWithoutCleanTableTest.groovy

```diff
@@ -0,0 +1,32 @@
+package com.github.vkorobkov.jfixtures
+
+import com.github.vkorobkov.jfixtures.testutil.YamlVirtualFolder
+import spock.lang.Specification
+
+import java.nio.file.Path
+
+class JFixturesWithoutCleanTableTest extends Specification implements YamlVirtualFolder {
```

Comment:
Let's test the these scenarios(if you didn't yet) in an integration test(using h2) and where the fixtures order does not matter. This test then could be removed